src/trytond_factories/attachment.py

```diff
@@ -0,0 +1,37 @@
+
+__all__ = [
+    'DataAttachment',
+    'LinkAttachment',
+]
+
+import factory
+import factory_trytond
+
+
+class _Attachment(factory_trytond.TrytonFactory):
+    class Meta:
+        model = 'ir.attachment'
+
+    name = factory.Faker('word')
+    type = factory.Faker('word', ext_word_list=['data', 'link'])
+    description = factory.Faker('sentence')
+    data = factory.Faker('binary', )
+    file_id = factory.Faker('word')
```

Comment:
Shouldn't data and file_id declarations be omitted so that the inherited LinkAttachment factory won't set those values?

Reply:
That's right. Those declarations have been wrongly survived!